test/e2e/tests/confirmations/signatures/personal-sign.spec.ts

```diff
@@ -0,0 +1,80 @@
+import { strict as assert } from 'assert';
+import { Suite } from 'mocha';
+import { withRedesignConfirmationFixtures } from '../helper-fixture';
+import {
+  DAPP_HOST_ADDRESS,
+  WINDOW_TITLES,
+  openDapp,
+  unlockWallet,
+} from '../../../helpers';
+import { Ganache } from '../../../seeder/ganache';
+import { Driver } from '../../../webdriver/driver';
+
+describe('Confirmation Signature - Personal Sign', function (this: Suite) {
+  if (!process.env.ENABLE_CONFIRMATION_REDESIGN) { return; }
+
+  it('initiates and confirms', async function () {
+    await withRedesignConfirmationFixtures(
+      this.test?.fullTitle(),
+      async ({ driver, ganacheServer }: { driver: Driver, ganacheServer: Ganache }) => {
+        const addresses = await ganacheServer.getAccounts();
+        const publicAddress = addresses?.[0] as string;
+
+        await unlockWallet(driver);
+        await openDapp(driver);
+        await driver.clickElement('#personalSign');
+        await driver.switchToWindowWithTitle(WINDOW_TITLES.Dialog);
+
+        await assertInfoValues(driver);
+
+        await driver.clickElement('[data-testid="confirm-footer-button"]');
```

Comment:
Minor, but is it worth creating a helper file in this directory to provide reusable functions such as `clickConfirmButton` as we do this in many of the tests?

It also helps to keep the test functions themselves high level and readable, plus it moves us slightly closer to page model objects that I'm a big fan of.

Reply:
hm.. unsure what helpers you are refering to. If it is what I am thinking - I am more of a fan of creating consts to reference the selectors unless the helper functions would involve more than one line

Reply:
I think that @matthewwalsh0 is hinting at some kind of page object model. We do not have any specific guidelines in the Extension... but there are some helper functions that are added [here](https://github.com/MetaMask/metamask-extension/blob/develop/test/e2e/helpers.js).

Reply:
As discussed, not urgent so can be done in future.